charts/merlin/templates/_config.tpl

```diff
@@ -0,0 +1,44 @@
+{{- define "merlin.renderedConfig" -}}
```

Comment:
`merlin.renderedConfig` is too broad. maybe `merlin.renderedImageBuilderConfig`?

Reply:
and since we call this file `_config.tpl` should we move here other related functions?

Reply:
that makes sense